admin/templates/configuration/etc/log4cxx.unittest.properties

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
It is convenient to set logging by module for logging (high severity in the module(s) you're testing/debugging and low severity elsewhere). This file is here as an example, it's not activated in the configuration, can I rename it to admin/templates/configuration/etc/log4cxx.example.properties and keep it please?